File: async-scala-client/src/main/scala/io/swagger/client/model/InstallmentComplete.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class InstallmentComplete (
  documentNumber: String,  // DOCNUMBER, ID OF THIS DOCUMENT FOR THIS transaction
date: DateTime,  // installment Due Date
grossValue: Double,  // Intallment amount, fraction of  ∑(lineAmount - lineTaxedDiscount)
netValue: Option[Double],  // net amount due for this installment, grossValue - ∑ (withhold amounts)
withholdingMode: Option[WithholdingMode],  // Inform if this payment is subject to Pis, Cofins or CSLL
withholdingPIS: Option[Double],  // calculated PIS-RF tax for this payment
withholdingCOFINS: Option[Double],  // calculated COFINS-RF tax for this payment
withholdingCSLL: Option[Double]  // calculated CSLL-RF tax for this payment
)

File: javascript-closure-angular-client/API/Client/InstallmentComplete.js
goog.provide('API.Client.InstallmentComplete');

/**
 * @record
 */
API.Client.InstallmentComplete = function() {}

/**
 * DOCNUMBER, ID OF THIS DOCUMENT FOR THIS transaction
 * @type {!string}
 * @export
 */
API.Client.InstallmentComplete.prototype.documentNumber;

/**
 * installment Due Date
 * @type {!Date}
 * @export
 */
API.Client.InstallmentComplete.prototype.date;

/**
 * Intallment amount, fraction of  ∑(lineAmount - lineTaxedDiscount)
 * @type {!number}
 * @export
 */
API.Client.InstallmentComplete.prototype.grossValue;

/**
 * net amount due for this installment, grossValue - ∑ (withhold amounts)
 * @type {!number}
 * @export
 */
API.Client.InstallmentComplete.prototype.netValue;

/**
 * Inform if this payment is subject to Pis, Cofins or CSLL
 * @type {!API.Client.WithholdingMode}
 * @export
 */
API.Client.InstallmentComplete.prototype.withholdingMode;

/**
 * calculated PIS-RF tax for this payment
 * @type {!number}
 * @export
 */
API.Client.InstallmentComplete.prototype.withholdingPIS;

/**
 * calculated COFINS-RF tax for this payment
 * @type {!number}
 * @export
 */
API.Client.InstallmentComplete.prototype.withholdingCOFINS;

/**
 * calculated CSLL-RF tax for this payment
 * @type {!number}
 * @export
 */
API.Client.InstallmentComplete.prototype.withholdingCSLL;

